src/library/browse/browsefeature.h

```diff
@@ -63,6 +63,8 @@ class BrowseFeature : public LibraryFeature {
     QStringList getDefaultQuickLinks() const;
     void saveQuickLinks();
     void loadQuickLinks();
+    // Get the list of devices (under "Removable Devices" section) on Linux.
+    QList<TreeItem*> getLinuxDevices();
```

Comment:
This function should be renamed to `getRemovableDevices()`. Considering the fact that the `this` pointer is just passed through it doesn't even need to be a member function. Move it into the .cpp file enclosed in an anonymous namespace and #ifdefs.

```
QList<TreeItem*> getRemovableDevices(LibraryFeature* pFeature)
```

Reply:
Done.

I renamed the function to `getLinuxRemovableDevices`. I think having `Linux` in the name of the function makes it clearer what the function does. Otherwise, a future reader of the code may think that the function works on all operating systems, whereas it's actually hardcoded with directory paths that are specific to Linux. Let me know if you still prefer that I change it.